rotkehlchen/exchanges/binance.py

```diff
@@ -358,9 +358,18 @@ def api_query(
                 except JSONDecodeError:
                     pass
 
+                if 'Invalid symbol' in msg and method == 'myTrades':
+                    assert options, 'We always provide options for myTrades call'
```

Comment:
nitpick: We always prefer to be explicit so I guess would be better
```suggestion
                    assert options is not None, 'We always provide options for myTrades call'
```

Reply:
True but I won't rerun all tests just for this.